composer.json

```
@@ -21,17 +21,17 @@
     "extra": {
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.3 || ^8.0",
```

Please use the version constrain like described in the related issue report #11:
```suggestion
        "php": "^7.3 || ~8.0.0",
```

---

Done. For my education, why not do it like is done for `^7.3` ?
What difference does it make?